jf_agent/main.py

```diff
@@ -15,14 +15,27 @@
 import urllib3
 import yaml
 
+import math
+
 from jf_agent import agent_logging, diagnostics, write_file
 from jf_agent.git import load_and_dump_git, get_git_client
-from jf_agent.jf_jira import get_basic_jira_connection, print_all_jira_fields, load_and_dump_jira
+from jf_agent.jf_jira import (
+    get_basic_jira_connection,
+    print_all_jira_fields,
+    load_and_dump_jira,
+    print_missing_repos_found_by_jira,
+)
 
 logger = logging.getLogger(__name__)
 
 JELLYFISH_API_BASE = 'https://jellyfish.co'
-VALID_RUN_MODES = ('download_and_send', 'download_only', 'send_only', 'print_all_jira_fields')
+VALID_RUN_MODES = (
+    'download_and_send',
+    'download_only',
+    'send_only',
+    'print_all_jira_fields',
+    'scan_jira_for_missing_repos',
```

Comment:
This run mode is in the same family as "print_all_jira_fields". It might be more understandable to the user if we called it something more parallel to that, something like "print_apparently_missing_git_repos".

Reply:
Also note the README.md should be updated for this new run mode